numba/core/byteflow.py

```diff
@@ -1088,6 +1182,35 @@ def _dup_topx(self, state, inst, count):
         for val in duped:
             state.push(val)
 
+    if PYVERSION in ((3, 12), ):
+        def op_CALL_INTRINSIC_1(self, state, inst):
+            # See https://github.com/python/cpython/blob/v3.12.0rc2/Include/
+            # internal/pycore_intrinsics.h#L3-L17C36
+            if inst.arg == 3:  # INTRINSIC_STOPITERATION_ERROR
+                state.append(inst, intrinsic="INTRINSIC_STOPITERATION_ERROR")
```

Comment:
Should the intrinsic be a string or an enum type given it's used again in `interpreter.py` in the corresponding opcode?

Reply:
Yes: 5071096b731c1108ad5a906a71e77783a6924f27